### ISSUE TITLE
Quote: "USDUSD" is missing

### ISSUE THREAD

**Issue by danielmarschall** [apilayer/currencylayer-API#16]
Hello,

I noticed that in CurrencyLayer, the quote entry "USDUSD" is missing. It should have the value 1.
My OpenSource software relied on this value and due to the change at CurrencyLayers, the apps don't work anymore.
Can you please re-add it again? Thank you.

Best regards,
Daniel Marschall

PS: I tried to contact via https://currencylayer.com/contact but I get the error message "Unable to create support ticket.".

**Comment by richardbiddle** [comment]
All conversions where source and destination are the same have been removed as well. It could be due to the issue mentioned above https://github.com/apilayer/currencylayer-API/issues/14 and https://github.com/apilayer/currencylayer-API/issues/15

**Comment by danielmarschall** [comment]
Of course, the entry "USDUSD" must always be 1 . The other bug reports reported that the value is not equal 1, which is clearly wrong. It is unfortunate for me that the bugfix was to remove "USDUSD" instead of hard-coding it to value 1. Backwards compatibility should be a number 1 goal for any API developer.

But I have added a workaround to my library. However, I hope there aren't too much of my users affected (my OpenSource middleware is 900 times downloaded. I have no idea who is actively using it, though).

Side note: I want to explain why the lack of "USDUSD" broke my code. I had to get a list of all currencies, and to get it, I simply listed all quotes and extracted the last 3 bytes of each entry. With "USDUSD" missing, I didn't got USD as currency.

**Comment by NikolaosWakem** [comment]
This random change just made my day not fun too!

**Comment by ikottman** [comment]
This is a breaking change to the API contract. The hardcoding of 1 should be done on the API side to maintain backwards compatibility.

**Comment by cyrnetix** [comment]
Please put USDUSD back with a constant value of 1. This a breaking change which screwed up my day and a lot of my colleagues.
